### PR TITLE
Sakattiy/aiops/docker

### PR DESCRIPTION
**Pull request by saleelpk**
Added ARG and markdownify packages to Dockerfile for enabling AIOps.

### CODE REVIEW

File: Docker/2.20 [hunk 1]
```diff
@@ -0,0 +1 @@
+2.12
```
Comment (MikeFrei) [inline Docker/2.20:1]: you should not be touching these files, I am assuming this is just showing up an an artifact of your not having pulled in changes?

File: Docker/2.12/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ RUN apt-get upgrade -y
 RUN apt-get install git -y
 RUN apt-get install git gcc libffi-dev g++ unixodbc-dev -y
 RUN pip3 install --upgrade pip
-RUN pip3 install msrestazure==0.6.4 hdbcli azure-storage==0.36.0 azure_storage_logging azure-mgmt-storage==16.0.0 azure-keyvault-secrets azure-identity prometheus_client retry pyodbc pandas zeep
+RUN pip3 install msrestazure==0.6.4 hdbcli azure-storage==0.36.0 azure_storage_logging azure-mgmt-storage==16.0.0 azure-keyvault-secrets azure-identity prometheus_client retry pyodbc pandas zeep azure-mgmt-resourcegraph azure-mgmt-resource markdownify
```
Comment (MikeFrei) [inline Docker/2.12/Dockerfile:15]: you all have tested this in a container build and deployment?  I assume these two mgmt packages do not have any dependencies on a newer version of the azure-storage package?

**Comment by MikeFrei** [comment]
please pull in latest changes from main and resend this PR since there are a lot of changes here that are not coming from you I assume.

File: Docker/2.12/Dockerfile (hunk 2 above)
Comment (saleelpk) [reply Docker/2.12/Dockerfile:15]: @MikeFrei. I remember you mentioning the importance of testing this with a container build and deployment. We have done that. The packages and their versions as mentioned here on line 15 works fine.

File: Docker/2.20 (hunk 1 above)
Comment (saleelpk) [reply Docker/2.20:1]: Let me have a call with you and update the docker files then.

Comment (saleelpk) [reply Docker/2.20]: Updated the proper file and pulled latest changes

**Comment by manikandankrishna** [comment]
Shouldn't these packages be added to this file as well
AzureMonitorForSAPSolutions/sapmon/setup/configureMonitorVM.sh

**Comment by saleelpk** [comment]
@manikandankrishna Is it required to update configorMonitorVM? Many packages are missing anyway.

**Review by MikeFrei — APPROVED** [review]

**Pull request merged**